### DISCUSSION TITLE
FHIR-typings

### DISCUSSION THREAD

**Discussion by kajami** [helsenorge/refero#157] [Q&A] [answered]
Hello,

I noticed that you use your own fhir-typings file (https://github.com/helsenorge/refero/blob/master/src/types/fhir.ts). Just wondering if there is some specific reason behind that fhir-typings file since those fhir-typings could also be imported from Definitely Typed package (https://www.npmjs.com/package/@types/fhir)? I noticed some differences between the typings when importing types from @types/fhir, even when the fhir-version is same.

**Comment by kennethmyhra** [comment, marked as answer]
Not sure when these types were first made available, from the commit history it seems they were first available May/June 2021. We started this project in 2017. Also, our types are auto-generated from the specification's XSD file(s).
